### PR TITLE
Check for Org id match to authorize

### PR DESCRIPTION
**Pull request by captDaylight**
- creates an `isAuthorized` value in store that checks if sdk org matches user org
- adds a `PhotonAuthorized` wrapper that displays children based on `isAuthorized` value

cc @samkotlove @cawfeecoder

### CODE REVIEW

File: packages/elements/src/photon-authorized/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  createEffect(() => {
+    setIsLoading(client?.authentication.state.isLoading || false);
+  });
+
+  return (
+    <>
+      <Show when={client && !authorized() && !isLoading()}>
+        <sl-alert variant="warning" open>
+          <sl-icon slot="icon" name="exclamation-triangle"></sl-icon>
+          <strong>You are not authorized to prescribe</strong>
```
Comment (captDaylight) [inline packages/elements/src/photon-authorized/index.tsx:30]: @samkotlove  update this to say something like please contact support at Photon?

File: packages/elements/src/photon-multirx-form/index.tsx [hunk 2]
```diff
@@ -331,14 +328,7 @@ customElement(
                 <sl-spinner style="font-size: 3rem;"></sl-spinner>
               </div>
             </Show>
-            <Show when={client && !authenticated() && hasError()}>
```
Comment (samkotlove) [inline packages/elements/src/photon-multirx-form/index.tsx:334]: This message was incorrect before (not being authenticated != not being authorized to prescribe), but i think we should show some error in this case

File: packages/elements/src/photon-authorized/index.tsx (hunk 1 above)
Comment (samkotlove) [reply packages/elements/src/photon-authorized/index.tsx:30]: Yeah I would keep it broad. "something went wrong" or w/e

File: packages/elements/src/photon-authorized/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  createEffect(() => {
+    setAuthenticated(client?.authentication.state.isAuthenticated || false);
+  });
+  createEffect(() => {
+    setAuthorized(client?.authentication.state.isAuthorized || false);
+  });
+
+  return (
+    <>
+      <Show when={client && !authenticated()}>
```
Comment (captDaylight) [inline packages/elements/src/photon-authorized/index.tsx:34]: @samkotlove this should handle the not authenticated message you mentioned below

Screen Shot 2023-02-20 at 12 33 38 PM

File: packages/elements/src/photon-authorized/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        <sl-alert variant="warning" open>
+          <sl-icon slot="icon" name="exclamation-triangle"></sl-icon>
+          <strong>You are not signed in</strong>
+          <br />
+        </sl-alert>
+      </Show>
+      <Show when={client && authenticated() && !authorized() && !isLoading()}>
+        <sl-alert variant="warning" open>
+          <sl-icon slot="icon" name="exclamation-triangle"></sl-icon>
+          <strong>Something went wrong, please contact support at Photon Health</strong>
```
Comment (samkotlove) [inline packages/elements/src/photon-authorized/index.tsx:44]: could we include support@photon.health

**Review by samkotlove — APPROVED** [review]

**Pull request merged**